### PR TITLE
Pairguru

### PR DESCRIPTION
**Pull request by cool-beetles**

### CODE REVIEW

File: app/assets/javascripts/comments.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+// Place all the behaviors and hooks related to the matching controller here.
```
Comment (mpiask) [inline app/assets/javascripts/comments.js:1]: Please don't leave unnecessary files in the repository.

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  
+  def create
+    @comment = current_user.comments.create(comment_params)
+    @comment.movie_id = params[:movie_id]
+
+    if @comment.save
+      redirect_to movie_path(@comment.movie_id) 
+    else
+      redirect_to movie_path(@comment.movie_id)
+    end
```
Comment (mpiask) [inline app/controllers/comments_controller.rb:12]: Why `if/else` if it returns the same?

File: app/controllers/comments_controller.rb [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  end
+
+  def destroy
+    @comment = current_user.comments.find(params[:id])
+    @comment.destroy
+
+    redirect_to movie_path(@comment.movie_id)
```
Comment (mpiask) [inline app/controllers/comments_controller.rb:19]: Is `@` necessary?

File: app/controllers/movies_controller.rb [hunk 4]
```diff
@@ -7,6 +7,8 @@ def index
 
   def show
     @movie = Movie.find(params[:id])
+    @comment = Comment.new(movie_id: params[:movie_id])
+    @users = User.all
```
Comment (mpiask) [inline app/controllers/movies_controller.rb:11]: What for?

File: app/models/user.rb [hunk 5]
```diff
@@ -22,6 +22,13 @@ class User < ApplicationRecord
...
          :recoverable, :rememberable, :trackable, :validatable, :confirmable
+  has_many :comments
 
   validates :phone_number, format: { with: /\A[+]?\d+(?>[- .]\d+)*\z/, allow_nil: true }
-end
+  
+  def self.top_10
+    ActiveRecord::Base.connection.execute("SELECT users.name, COUNT(user_id) AS comments_count FROM users
+      INNER JOIN comments ON users.id = comments.user_id WHERE comments.created_at > '#{(Time.now - 7.days).to_formatted_s(:db)}' 
+      GROUP BY name ORDER BY comments_count DESC LIMIt 10")
```
Comment (mpiask) [inline app/models/user.rb:32]: Typo in `LIMIt`. ;)

File: spec/models/comment_spec.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (mpiask) [inline spec/models/comment_spec.rb:4]: Maybe add some tests instead of leaving an empty file. :)

**Review by mpiask — COMMENTED** [review]
Hey!
It's not bad, but it needs some fixes. I have added a few comments.
Moreover please reread rules in readme: your solution lacks tests!
Will you be able to amend it during the following days?

**Comment by cool-beetles** [comment]
Hi @mpiask,
Thank you for a quick review. I improved typo and removed @, wrote tests for model. I need create tests for comments_controller. So, I'll try send my solution till Wednesday end of day. Are you OK with that?

**Comment by cool-beetles** [comment]
Hi @mpiask, 
Here is link for PR after changes:
https://github.com/cool-beetles/pairguru/pull/2

I had a problem with tests. I didn't use FactoryBot before, so I tried around. 
I'm waiting for your answer.
